cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -225,10 +226,11 @@ describe("Timeline", () => {
             );
 
             // Wait until configuration is finished
-            cy.contains(
-                ".mx_RoomView_body .mx_GenericEventListSummary .mx_GenericEventListSummary_summary",
-                "created and configured the room.",
-            ).should("exist");
+            cy.get(".mx_RoomView_body .mx_GenericEventListSummary[data-layout='group']").within(() => {
+                cy.get(".mx_GenericEventListSummary_summary")
```

Comment:
I would personally remove `get(".mx_GenericEventListSummary_summary")` . This query does already run „within“ `.mx_GenericEventListSummary`.

→

```
cy.fndByText(OLD_NAME + " created and configured the room.")
```